virt-v2v/cold/entrypoint.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const (
+	OVA     = "ova"
+	vSphere = "vSphere"
+	DIR     = "/var/tmp/v2v"
+	FS      = "/mnt/disks/disk[0-9]*"
+	Block   = "/dev/block[0-9]*"
+	VDDK    = "/opt/vmware-vix-disklib-distrib"
+)
+
+var (
+	xmlFilePath string
+	server      *http.Server
+)
+
+func main() {
+	virtV2vArgs := []string{"virt-v2v", "-v", "-x"}
+	source := os.Getenv("V2V_source")
+
+	requiredEnvVars := map[string][]string{
+		vSphere: {"V2V_libvirtURL", "V2V_secretKey", "V2V_vmName"},
+		OVA:     {"V2V_diskPath", "V2V_vmName"},
+	}
+
+	if envVars, ok := requiredEnvVars[source]; ok {
+		if !checkEnvVariablesSet(envVars...) {
+			fmt.Printf("Following environment variables need to be defined: %v\n", envVars)
+			os.Exit(1)
```

Comment:
how about using `klog.Fatal` here instead?

Reply:
klog isnt working here because virt-v2v is set in a different  workspace so this dependencies are not recognized here, that the reason I went with simple print. We consider extracting this part to the main workspace like we do for the monitor but we agreed its to big of a change all at once, so we can do it later.

Reply:
ok